themes/bootstrap3/templates/RecordDriver/SolrOverdrive/core.phtml

```diff
@@ -132,18 +132,18 @@ elseif ($avail->data):
 
     if($button):
         $button1Markup = <<<BTN1MARKUP
-            <a class="btn btn-primary {$button->class}" 
-               data-lightbox title="{$button->text}" 
-               href="{$button->url}"><i class="fa fa-{$button->icon}" aria-hidden="true"></i>&nbsp; 
+            <a class="btn btn-primary {$button->class}"
+               data-lightbox title="{$button->text}"
+               href="{$button->url}"><?=$this->icon('{$button->icon}') ?>&nbsp;
```

Comment:
I think this file needs significantly more work -- all the possible values of `$button->icon` don't seem to be accounted for in the icon file (unless I'm missing/misunderstanding something).

I realize that this is more or less impossible for us to test, but I want to be careful not to break anything, and I think as revised here, this is going to cause problems.

Reply:
It look like these are the values that are possible:

```php
$button->icon = "arrow-left" (od_but_checkout)
$button->icon = "flag" (od_but_cancel_hold)
$button->icon = "flag" (od_but_hold)
$button->icon = "sign-in" (od_but_return)
$button2->icon = "download" (od_but_gettitle)
```

I agree these need to be changed to `overdrive-` prefixed icons. We have download and return. We'll need.

```
overdrive-cancel-hold (FontAwesome:flag-o)
overdrive-checkout (FontAwesome:arrow-left)
overdrive-checkout-rtl (FontAwesome:arrow-right)
overdrive-sign-in (Alias:sign-in)
```

We might get clearer semantics if we switch hold icons across the board to a bell, so we can use bell-slash to cancel holds:

![image](https://user-images.githubusercontent.com/451107/157071687-54a43c80-5e72-4a42-8576-a2fa0fbefbb4.png)

Reply:
Maybe unrelated, but the arrows for borrow and return are not great. I don't have a better replacement in FA4.

Also, we might get clearer semantics if we switch hold icons across the board to a bell, so we can use bell-slash to cancel holds:

![image](https://user-images.githubusercontent.com/451107/157071687-54a43c80-5e72-4a42-8576-a2fa0fbefbb4.png)

Reply:
@crhallberg, what icons are we currently using for canceling holds, or is there no such thing at the moment? I'd prefer to keep icons consistent with past precedent while refactoring -- I think it's better to change them as a separate pass if we can to keep plumbing separate from design, so to speak. I feel like the bell icon is so associated with notifications that it might not be the best fit here (not that the flag makes a whole lot more sense). I wonder if a generic cancel icon might suffice, though (i.e. slashed circle).

Reply:
Right now it's all flags. For holds, a bell makes sense because you're being notified when the book is available? This is all future consideration of course.

Reply:
Ahh, yes, I see what you mean regarding bell for holds. That might work, as long as we can fix the issue here with all the different types of requests getting their icons mixed up by the Javascript code -- I guess that's the first order of business!